dev.aunetx.deezer.yml

```diff
@@ -0,0 +1,62 @@
+app-id: dev.aunetx.deezer
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+tags: ["proprietary"]
+finish-args:
+  # Basic permissions
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  # Allow other instances to see lockfiles
+  - --env=TMPDIR=/var/tmp
+  # DBus permissions
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.gnome.SettingsDaemon
+  - --talk-name=org.gnome.SessionManager
+  - --own-name=org.mpris.MediaPlayer2.chromium.*
+  # Fs access: music/downloads to send mp3, pictures to change profile picture
+  - --filesystem=xdg-music:ro
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-download:ro
+modules:
+  - name: deezer
+    buildsystem: simple
+    sources:
+      # The application datas
+      - type: file
+        path: dev.aunetx.deezer.appdata.xml
+        dest: extra
+      - type: file
+        path: dev.aunetx.deezer.svg
+        dest: extra
+      - type: file
+        path: dev.aunetx.deezer.desktop
+        dest: extra
+      # Wrapper to launch the app
+      - type: script
+        dest-filename: run.sh
+        commands:
+          - zypak-wrapper /app/main/deezer-desktop "$@"
+      - type: archive
+        url: https://github.com/aunetx/deezer-linux/releases/download/v5.30.570-6/deezer-desktop-5.30.570-x64.tar.xz
```

Comment:
I see the appstram icone and desktop file in the upstream repo. So why are they in this repository?

Reply:
Actually, neither the icon nor the desktop file are included in the builds that are produced:
- the icon is only used when producing actual packages (deb, rpm, ...) but not when producing the `tar.gz` archive
- the desktop file is not packaged at all as it was exclusively for the flatpaks produced in the repository before that, and I will entirely remove this repository's flatpak repo once it is hosted on flathub (so the desktop file needs to be hosted here)

Reply:
given that it's better when the file is managed upstream (it is absolutely NOT specific to flatpak) keeping it upstream is the way to go.